<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_sync</class>
 <widget class="QDialog" name="Dialog_sync">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_rossler">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Rossler Attractor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_arnold">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Arnolds Cat Map</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sync">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>441</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Desktop/bg.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>0</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>341</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>341</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_sync</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
